Holovibes/sources/thread/frame_record_worker.cc

```diff
@@ -43,6 +43,17 @@ size_t FrameRecordWorker::compute_fps_average() const
     return ret;
 }
 
+std::string prepend_date_to_filepath(std::string record_file_path)
+{
+    //? Do we move this to the export panel, and consider the date to be set when the path is set/on startup ?
+    std::filesystem::path filePath(record_file_path);
+    std::string date = Chrono::get_current_date();
+    std::string filename = filePath.filename().string();
+    std::string path = filePath.parent_path().string();
+    std::filesystem::path newFilePath = path + "/" + date + "_" + filename;
+    return newFilePath.string();
+}
+
 std::string prepend_R_to_filename(std::string record_file_path)
```

Comment:
Maybe create a wrapper function since both functions does the same thing. Something like:
```cpp
std::string prepend_string_to_filepath(std::string file_path, std::string str)
{
    std::filesystem::path filePath(file_path);
    std::string filename = filePath.filename().string();
    std::string path = filePath.parent_path().string();
    std::filesystem::path newFilePath = path + "/" + str + "_" + filename;
    return newFilePath.string();
}
```